app/models/harvest.rb

```diff
@@ -7,6 +7,7 @@ class Harvest < ActiveRecord::Base
   belongs_to :crop
```

Comment:
No, but I think this is the least worst option with our current datamodel. 

I do question, long term, why track harvests of thing you do to have planted in your garden?